<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>reportDialogClass</class>
 <widget class="QDialog" name="reportDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>reportDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Od:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="fromDate">
         <property name="date">
          <date>
           <year>2023</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Do:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="toDate">
         <property name="date">
          <date>
           <year>2024</year>
           <month>1</month>
           <day>1</day>
          </date>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="generateButton">
         <property name="text">
          <string>Generuj raport</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="okButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="page"/>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>reportDialogClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>524</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel">
     <x>425</x>
     <y>9</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
